pycheribuild/projects/syzkaller.py

```diff
@@ -62,6 +67,13 @@ def setup_config_options(cls, **kwargs):
             "run-sysgen", show_help=True,
             help="Rerun syz-extract and syz-sysgen to rebuild generated Go "
                  "syscall descriptions.")
+        cls.use_go = cls.add_config_option("use-go", kind=GOROOTOptions,
+            default=GOROOTOptions.DEFAULT, enum_choice_strings=[t.value for t in GOROOTOptions],
+            help="Specify GOROOT default is system")
+
+    def create_new_path(self, config):
+        self._new_path = (str(self.config.cheri_sdk_dir / "bin") + ":" +
+                          str(self.config.dollar_path_with_other_tools))
```

Comment:
Instead of setting the value, I'd return it. Then, instead of creating a field `_new_path` and maintaining it, we can simply call `get_path()` when we construct `args`.